npm/commander/2.19.0/@n4jsd/commander/index.n4jsd

```diff
@@ -0,0 +1,19 @@
+/**
+ * @see https://github.com/tj/commander.js/
+ */
+export external public class __Command {
+    public version(v: string, customFlag: string=): __Command;
+    public arguments(args: string): __Command
+    public description(desc: string): __Command;
+    public usage(usg: string): __Command
+    public option(opt: string, descr: string, fn: Function | string=): __Command;
+    public command(cmd: string, descr: string, config: ~Object with {isDefault: boolean}): __Command;
+    public alias(alias: string): __Command;
+    public action(fn: Function): __Command;
+    public parse(argv: [string]): Object;
+    public args: [string];
+    public help: Function;
+}
+// By default, export an instance of Command
+export default external public const program: __Command;
+export external public const Command: constructor{__Command};
```

Comment:
So you don't need this, if you export the class above.

Reply:
done